plugins/modules/synchronize.py

```diff
@@ -75,7 +75,7 @@
     description:
       - Delete files in C(dest) that don't exist (after transfer, not before) in the C(src) path.
       - This option requires C(recursive=yes).
```

Comment:
And while I'm at it:
```suggestion
      - Delete files in I(dest) that don't exist (after transfer, not before) in the I(src) path.
      - This option requires I(recursive=yes).
```